api/src/main/java/jakarta/mail/Session.java

```diff
@@ -198,6 +198,7 @@
  * @author Max Spivak
  */
 
+@aQute.bnd.annotation.spi.ServiceConsumer(value = Provider.class)
```

Comment:
There is currently a restriction on using 3rd party libraries in the APIs and only (non Java SE/Jakarta EE packages can be used.